docs/testnet-guardian-runbook.md

```diff
@@ -0,0 +1,130 @@
+# Testnet-Guardian Runbook
+
+## Overview
+
+A testnet guardian is the point of contact for anything engineering-related for an upcoming or a running testnet. Broadly speaking, a testnet guardian would perform/coordinate tasks involving building, testing and deploying testnet(s) for a release. They would also serve as a point-of-contact for protocol/product related queries from the community. This document describes all the tasks that a testnet guardian would perform during a release cycle and describes possible actions  in the case of events that affect the testnet. The role of a testnet guardian spans over a release cycle starting from code-freeze from the engineering team to the end of the release. The initial phase between code freeze and the public release is the QA phase where the testnet guardian will be responsible for building, deploying and testing internal QA-nets. On successful completion of the QA phase, the release phase starts during which the testnet guardian would support the release process and be the point of contact for the community.
+
+## QA-net
+
+The role of a testnet guardian for a release begins after the code-freeze. The following is the list of tasks that a testnet guardian would be responsible for:
+
+### Release branch
+
+Cut a release branch after the code-freeze(The naming convention for beta-testnets is `release/0.0.<release_number>-beta`. For example, `release/0.0.13-beta`)
+
+### Build
+
+The latest commit on the release branch should have successful builds for both macOS and Linux.
+Go to the CI workflow for the specific commit and look for `build- macos` and `build-artifacts--testnet_postake_medium_curves` jobs for mac and linux builds respectively.
+
+##### Update brew package
```

Comment:
These sections for updating the brew and apt packages should be done after the QA net is validated when we actually do the release. We don't typically do these as part of the QA net right now, afaik.